src/shared/case-editor/case-progress.component.spec.ts

```diff
@@ -0,0 +1,271 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { DebugElement, NO_ERRORS_SCHEMA, Component, Input, EventEmitter, Output } from '@angular/core';
+import createSpyObj = jasmine.createSpyObj;
+import { CasesService } from '../cases/cases.service';
+import { CaseProgressComponent } from './case-progress.component';
+import { CaseEventTrigger, Draft, CaseDetails, CaseEventData, CaseView } from '../domain';
+import { createCaseEventTrigger } from '../fixture/shared.fixture';
+import { AlertService } from '../alert';
+import { of, Observable } from 'rxjs';
+import { HttpError } from '../http';
+
+@Component({
+  selector: 'ccd-case-edit',
+  template: ``
+})
+class CaseEditComponent {
```

Comment:
Can't a mock component be used here instead of an actual one?

Reply:
See above.